### PR TITLE
added model card details comparing cifar to imagenet resnets

### PR DESCRIPTION
**Pull request by growlix**

### CODE REVIEW

File: composer/models/resnet/README.md [hunk 1]
```diff
@@ -37,6 +37,9 @@ ResNet family members are defined by their number of layers. Parameter count, ac
 | ResNet-101           | 44.5M           | 78.1%        | 5.50 hrs                 |
 | ResNet-152           | 60.2M           | TBA          | TBA                      |
 
+
+> ❗ **Note**: Please see the :doc:`/model_cards/cifar_resnet` model card for the differences between CIFAR and ImageNet ResNets.
```
Comment (hanlint) [inline composer/models/resnet/README.md:41]: ah, this won't work since this is a README file -- you'll have to hard link the URL

**Review by hanlint — APPROVED** [review]

File: composer/models/resnet/README.md (hunk 1 above)
Comment (growlix) [reply composer/models/resnet/README.md:41]: What's the proper url? Is it https://docs.mosaicml.com/en/v0.5.0/model_cards/cifar_resnet.html, or is there a way to make it version-independent (e.g. change v0.5.0 to something else)?

Comment (ravi-mosaicml) [reply composer/models/resnet/README.md:41]: `https://docs.mosaicml.com/en/stable/model_cards/cifar_resnet.html`

**Review by ravi-mosaicml — APPROVED** [review]
LGTM

**Pull request merged**